demux/demux_lavf.c

```diff
@@ -17,7 +17,6 @@
  * License along with mpv.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include <pthread.h>
```

Comment:
Doesn't seem to need pthreads either.

Reply:
See https://github.com/mpv-player/mpv/pull/12709#issuecomment-1793502512 for explenation why some stray `<pthread.h>` has to be there until new win32 impl.

Reply:
removed